src/features/header-flyout/HeaderFlyout.js

```diff
@@ -27,43 +29,44 @@ class HeaderFlyout extends React.Component<Props> {
 
     static defaultProps = {
         position: 'bottom-left',
+        isResponsive: false,
     };
 
     render() {
-        const { header, footer, flyoutButton, children, scrollRefFn, className, ...rest } = this.props;
+        const { header, footer, isResponsive, children, scrollRefFn, className, ...rest } = this.props;
 
         return (
-            <Flyout
+            <ResponsiveFlyout
```

Comment:
It may be safer to move this refactor to a separate PR to ease potential reverts.